tgmc.dme

```diff
@@ -998,6 +780,47 @@
 #include "code\modules\mapping\space_management\space_reservation.dm"
 #include "code\modules\mapping\space_management\traits.dm"
 #include "code\modules\mapping\space_management\zlevel_manager.dm"
+#include "code\modules\marines\cosmetics.dm"
+#include "code\modules\marines\dropship_part_fabricator.dm"
+#include "code\modules\marines\gift_wrappaper.dm"
+#include "code\modules\marines\line_nexter.dm"
+#include "code\modules\marines\maps.dm"
+#include "code\modules\marines\memorial.dm"
+#include "code\modules\marines\mortar.dm"
+#include "code\modules\marines\overwatch.dm"
+#include "code\modules\marines\self_destruct.dm"
+#include "code\modules\marines\equipment\binoculars.dm"
+#include "code\modules\marines\equipment\marine_gear.dm"
+#include "code\modules\marines\equipment\motion_detector.dm"
+#include "code\modules\marines\equipment\portable_vendor.dm"
+#include "code\modules\marines\weapons\dropship_ammo.dm"

```

Comment:
`marines` is a horrible module. You can put either everything or nothing it in.
Dropship stuff warrants its own module.